app/code/UpdateDocsCronTask.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+class UpdateDocsCronTask implements CronTask {
```

Comment:
@SpiritLevel fancy making a pull request against the platform branch to change this please :) I'm across many things today so that would help. :D